tests/consensus_tests/test_replicate_points.py

```
@@ -51,7 +51,7 @@ def test_replicate_points_stream_transfer(tmp_path: pathlib.Path):
     assert_project_root()
 
     # seed port to reuse the same port for the restarted nodes
-    peer_api_uris, peer_dirs, bootstrap_uri = start_cluster(tmp_path, N_PEERS, 20000)
+    peer_api_uris, peer_dirs, bootstrap_uri = start_cluster(tmp_path, N_PEERS)
```

Ruff (0.15.10)

[warning] Unpacked variable `peer_dirs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)